exercises/random_number.cpp

```diff
@@ -4,3 +4,9 @@
   Output:
   The random number is: 4
 */
+#include <iostream>
+using namespace std;
+int main(){
+  srand(time(NULL));
+  cout<<"The random number is "<<rand()%10+1<<"\n";
+}
```

Comment:
in realtà dovevi fare due PR, una per esercizio

la randomizzazione del numero in una funzione a parte